stable/index.yaml

```diff
@@ -14,9 +14,31 @@ entries:
     - https://raw.githubusercontent.com/portworx/helm/master/stable/portworx-1.0.0.tgz
     version: 1.0.0
   px-backup:
+  - apiVersion: v1
+    appVersion: 1.1.0-rc1
+    created: "2020-08-19T22:26:45.978568+05:30"
+    description: A Helm chart for installing PX-Backup with PX-Central on Kubernetes
+      and Openshift
+    digest: 4e105ef49db6ffb1a59c24ed0d306878eddf4752d31624e91f6cf2b39eb73303
+    home: https://portworx.com/
+    icon: https://raw.githubusercontent.com/portworx/helm/master/doc/media/px-logo.png
+    keywords:
+    - px-backup
+    - storage
+    - persistent disk
+    - pvc
+    - portworx
+    - px-central
+    - lighthouse
+    name: px-backup
+    sources:
+    - https://github.com/portworx/helm/tree/master/charts/px-backup
+    urls:
+    - https://raw.githubusercontent.com/portworx/helm/master/stable/px-backup-1.1.0.tgz
+    version: 1.1.0
   - apiVersion: v1
     appVersion: 1.0.2
-    created: "2020-08-19T12:57:03.127424+05:30"
+    created: "2020-08-19T22:26:45.977443+05:30"
```

Comment:
Should these be changing?

Reply:
Yes, 
`helm package ../charts/px-backup/ ../charts/px-license-server/`
`helm repo index --url https://raw.githubusercontent.com/portworx/helm/master/stable/ .`

this command updates the time for the package but not actually digest.